src/model/transferlog.js

```diff
@@ -102,21 +143,20 @@ module.exports = class TransferLog {
     return true
   }
 
-  * _storePackaged (packaged) {
-    this._storeInCache(packaged)
-    this._store.put('transfer_' + packaged.transfer.id, JSON.stringify(packaged))
+  _storePackaged (packaged) {
+    this._put(packaged.transfer.id, JSON.stringify(packaged))
```

Comment:
can we be sure this `.id` is unique? What if the other peer messes up and reuses an id, the old transfer will be erased from history on the next restart, don't we want to safeguard against that? Maybe always generate the id ourselves, we can also just use incremental integers, right?